CHANGELOG.md

```diff
@@ -12,6 +14,25 @@
 - Replaced the example for Java system properties in `specification/configuration.md`.
 - Increase profiling rate from 1s to 10s.
 
+### Configuration
+
```

Comment:
According to https://github.com/signalfx/gdi-specification/blob/main/specification/repository.md#required-files 
we should have `Breaking changes` / `Bug fixes` / `Enhancements` subsections

Reply:
> Each release SHOULD contain separate sections for each major functionality area (if applicable). The following sub-sections MAY be used, as appropriate or specified.

Which subsection do you think is appropriate? It didn't seem to fit one but rather refer to ta general change of the section.

Reply:
See https://github.com/signalfx/gdi-specification/pull/167/files#r857342834 

Feel free to resolve/change/ignore based on your judgement